tests/periph_flashpage_unittest/Kconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config APPLICATION
+    bool
+    default y
+    depends on TEST_KCONFIG
```

Comment:
Had an (unrelated) chat with @leandrolanzieri about Kconfig migration and this file does not seem to be necessary (and actually could complicate things).

Reply:
Can you elaborate on why it would complicate things? Or maybe @leandrolanzieri can?

Reply:
(I agree it is not needed, copy pasta)

Reply:
I think it is just not needed. In fact it would still pass the test as it is defaults to y.